src/css/metacatui-common.css

```diff
@@ -6278,6 +6293,41 @@ body.mapMode{
 .filter input[type='text']{
 	width: 80%;
 }
+
+.filter.annotation-filter {
+	min-width: 300px;
+}
+
+/*
+	Overrides for default Bootstrap styling for the searchable select
+	component when we use it as a Portal data search filter. These overrides make
+	the component look nearly identical to other filters (that use native form
+	elements)
+*/
+.filter.annotation-filter .ui.fluid.dropdown>.dropdown.icon {
+	top: 25%;
+}
+
+.filter.annotation-filter .dropdown.search {
+	padding: 0;
+	font-size: 100%;
+	box-sizing: border-box;
+	border-color: #ccc;
+	border-radius: 4px;
+}
+
+.filter.annotation-filter .dropdown.multiple.ui {
+	padding: 0;
+}
+
+.filter.annotation-filter .ui.multiple.search.dropdown > input.search {
+	font-size: inherit;
+	line-height: 1em;
+	min-height: 1em;
+	margin: 0.4em;
+}
+/* End Overrides for default Bootstrap styling for the searchable select */
+
```

Comment:
Just a note to look this over, @laurenwalker. There's a matching commit that explains more of the rationale too. I opted for this approach instead of refactoring the SearchableSelect styling.